machine/xen/xenstore_client.go

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2022, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+//go:build xen
+// +build xen
+
+package xen
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"kraftkit.sh/log"
+	"xenbits.xenproject.org/git-http/xen.git/tools/golang/xenlight"
+)
+
+type XenstoreOperation uint32
+
+const (
+	WatchOp    XenstoreOperation = 4
+	UnwatchOp  XenstoreOperation = 5
+	WatchEvent XenstoreOperation = 15
+	Error      XenstoreOperation = 16
```

Comment:
Are these bit flags? if yes you could use `0b...` to specify them

Reply:
These are request/response codes for `xenstore`, not flags.